### ISSUE TITLE
UI Update - Welcome Cards (Text Weight/Color, Button text, background color, top nav shadow, next arrow vertical alignment, active top nav state)

### ISSUE THREAD

**Issue by Watson-B** [ezl/dailyidea.com#869]
I want to document some UI items to eventually update that I've noticed throughout the site. None of these are critical priority, but would be "nice-to-have updates" at some point. If there is a better way to organize these, just let me know.

Current:

![Ui-color](https://user-images.githubusercontent.com/65913664/105255728-00d2a380-5b39-11eb-865a-21285de1331d.png)

Design:

![ui+color2](https://user-images.githubusercontent.com/65913664/105255737-08924800-5b39-11eb-801c-169235e4af89.png)

Background color: #F8F8FA;
Default Text color: 4C4763;
Subdued (gray) text: HSLA (251, 16, 33, .6);
Header: Avenir LT Std (weight: 65 Medium; size: 28;);
Body: Avenir LT Std (weight: 45 Book; size: 18;);
Button Text: Avenir LT Std (weight: 65 Medium; size: 14;);

Can we set alignment for the next/previous arrows to be centered vertically to the card?
I think there is another ticket where we talked about this, but I'm not seeing it now. Can we set Explore to active state when the use lands on the welcome cards, and set the inactive top nav tabs to match the normal inactive states.
Update vertical padding on the cards to 40px. 
Add drop shadow to the top nav.

**Comment by ikosovec** [comment]
I think these cosmetical updates can be put to be worked on next. These do make the whole look & feel better.

**Comment by ezl** [comment]
i support it.

**Comment by vedmant** [comment]
@Watson-B  Few questions.

> Can we set Explore to active state when the use lands on the welcome cards.

Ok, but it might be confusing, cause home page and explore pages are different pages, this cards we have on home page with url "/", and explore urls are "/i/sdfs/some-idea", that represent ideas swiper. If I set explore active on home page then it may appear that Explore link is not clickable, but it is and leads on explore page. What do you think about this @ezl ?

> , and set the inactive top nav tabs to match the normal inactive states.

What do you mean by normal inactive states? 

> Update vertical padding on the cards to 40px.

The problem is that on design card height is different from actual height in browser, so I made it stretch to fit to card. Can you share design with the size of the card matching size in the browser, plus a narrow variant, so I can understand how it shuold scale vertically and horizontally (including different aspect ratios like tall and narrow, for example: https://prnt.sc/yqesit https://prnt.sc/yqethh and very large screen: https://prnt.sc/yqeuqs)

**Comment by vedmant** [comment]
TODO 

- [ ] Set alignment for the next/previous arrows to be centered vertically to the card
- [ ] Add drop shadow to the top nav.
- [ ] Different paddings?
- [ ] Explore menu item active?

**Comment by Watson-B** [comment]
@vedmant re: Active Explore state; The point you make about making the explore link active makes it seem like it's not clickable to skip the welcome is a good point. I guess the confusion I'm having then stems from me conceptualizing the Welcome cards as a conditional state to put at the front of the explore flow when the user isn't logged in rather than the Welcome Cards being the "Home". So after viewing them, when the user clicks the logo they would be taken to the top of the Explore stack. Currently, when you are logged in (using the beta site) and click the logo in the top left, it returns you to the "welcome screen" with the "get started" buttons, which makes it seem like the user is logged out, even though clicking "my ideas" will still take them back to their profile.  @ikosovec @ezl What are your thoughts on the flow for clicking the logo in the top left?

clicking the logo from "My Ideas"
![image](https://user-images.githubusercontent.com/65913664/107415055-bdc87800-6ac7-11eb-9bb4-5851f514c670.png)
Takes you back to the welcome screen
![image](https://user-images.githubusercontent.com/65913664/107415103-cd47c100-6ac7-11eb-8615-1095c4c7caa1.png)

**Comment by Watson-B** [comment]
Re: the inactive states
It looks like it's been updated already on the beta site, I was referring to how on some pages the inactive states are black links and others gray.

**Comment by Watson-B** [comment]
> The problem is that on design card height is different from actual height in browser, so I made it stretch to fit to card. Can you share design with the size of the card matching size in the browser, plus a narrow variant, so I can understand how it shuold scale vertically and horizontally (including different aspect ratios like tall and narrow, for example: https://prnt.sc/yqesit https://prnt.sc/yqethh and very large screen: https://prnt.sc/yqeuqs)

@vedmant Just want to clarify before I work on this, I'm going to take the screenshots you sent and using the respective card size in each, show how the design scales, correct?

**Comment by vedmant** [comment]
@Watson-B  

> Re: the inactive states
It looks like it's been updated already on the beta site, I was referring to how on some pages the inactive states are black links and others gray.

They are grey when user logged out.

> @vedmant Just want to clarify before I work on this, I'm going to take the screenshots you sent and using the respective card size in each, show how the design scales, correct?

Yes, to understand how it scales and also how it looks in different aspect ratios (like 2 to 1 or 1 to 1), as card can be wide and short in height (content can be scrollable also if needed) or can be almost square.

**Comment by Watson-B** [comment]
@vedmant Sounds good, I will do that this afternoon.

**Comment by vedmant** [comment]
@Watson-B  It was a task previously to make them gray when logged out, I think @ezl asked for this.

**Comment by Watson-B** [comment]
@vedmant Mocked each [in Figma](https://www.figma.com/file/VTAWAu8G6jtzpWYulfpw9m/Daily-Idea?node-id=576%3A2). Guessing the linked images were downloading at 2x, so did 1x on the left and 2x on the right. The first two linked images look good, it's really just the "very-large screen"; where the headline, subheader, and button move away from the top left, the arrows away from the bottom right, and the gif doesn't cover the full width. In the mock I did, I fixed the header to the top left and scaled it up a bit.  

![image](https://user-images.githubusercontent.com/65913664/107432047-36d1ca80-6adc-11eb-9c38-849ceef3e966.png)

In regards to font scaling, is it an issue that the smallest screen has a vertical ratio to the "very-large screen" of 1.85, and yet I'm only scaling the Header up by a ratio of 1.42 across the same screens?
